core-bundle/src/Controller/RedirectController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Controller;
+
+use Contao\PageModel;
+use Symfony\Bundle\FrameworkBundle\Controller\RedirectController as SymfonyRedirectController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @internal
+ */
+class RedirectController extends SymfonyRedirectController
```

Comment:
The `SymfonyRedirectController` is `@final`, you are not allowed to extend it. You should inject it in your own service. Maybe then we can extend our `AbstractController` to re-use the `getPageModel()` method?

Reply:
Changed in 6e94e5772a7f12577e4e63dfae830b6557995ded. We cannot use `AbstractFragmentController::getPageModel` though, because
 a) we're not a fragment controller
 b) we **must** use the request that is provided in the action, not some "current request"